vane-trifles/src/main/java/org/oddlama/vane/trifles/items/storage/Pouch.java

```diff
@@ -60,9 +144,14 @@ public void on_player_right_click(final PlayerInteractEvent event) {
         event.setUseInteractedBlock(Event.Result.DENY);
         event.setUseItemInHand(Event.Result.DENY);
 
+        ItemMeta pouchMeta = item.getItemMeta();
+
         if (get_module().storage_group.open_block_state_inventory(player, item)) {
             player.getWorld().playSound(player, Sound.ITEM_BUNDLE_DROP_CONTENTS, 1.0f, 1.2f);
             swing_arm(player, event.getHand());
+            pouchMeta.setLore(List.of(openedText));
```

Comment:
I think we need to change how lore is handled here. This still overwrites lore of other plugins, so we need to make sure to tag our lore and leave anything else as-is. We've done that a couple of times in vane by using a sentinel that we can use to identify which lines belong to our plugin. Then we only remove those and add new lines with that sentinel. You can have a look at the Trowel implementation to see how it works:

https://github.com/oddlama/vane/blob/17835a82f0c2d1cbdf6607ed3eba46c528b87504/vane-trifles/src/main/java/org/oddlama/vane/trifles/items/Trowel.java#L135-L150